example/tests/test9.sh

```diff
@@ -0,0 +1,20 @@
+wd=test9
+
+if [ -d $wd ]; then
+    rm -r $wd
+fi
+
+# The expected runtime of this test is ~20 minutes.
```

Comment:
Runtime mismatch between this line and the test description in the README file (which states ~35 minutes).

Reply:
I fixed this.